packages/babel-compat-data/scripts/download-compat-table.sh

```diff
@@ -16,5 +16,5 @@ fi
 
 rm -rf build/compat-table
 mkdir -p build
-git clone --branch=gh-pages --single-branch --shallow-since=2020-04-20 https://github.com/kangax/compat-table.git build/compat-table
+git clone --branch=gh-pages --single-branch --shallow-since=2020-04-01 https://github.com/kangax/compat-table.git build/compat-table
```

Comment:
I think this is a bug of GitHub.

```
$ git show --pretty=fuller HEAD

commit 9e07df3875d8416af85cf523716519e9dd1e5e44 (HEAD -> gh-pages, upstream/gh-pages, upstream/HEAD)
Author:     Nicolò Ribaudo <nicolo.ribaudo@gmail.com>
AuthorDate: Fri Apr 17 15:06:37 2020 +0200
Commit:     Jordan Harband <ljharb@gmail.com>
CommitDate: Fri Apr 17 15:06:37 2020 +0200
```

The CommitDate should be the time when the PR is merged instead of when the PR is authored. Note that`--shallow-since` filters history by `CommitDate` so it should have been included in `--shallow-since=2020-04-20` if the `gh-pages` are indeed chronological.

```
$ git rev-list HEAD --since=2020-04-20
(empty)

$ git rev-list HEAD~ --since=2020-04-20

0243431898be45cf53aed6c5b7332265cf7e69a8
6e9e589e966ec4b69ea35c63d55cf7d800ee94b6
989fad76f621c260f304d75c07c386c4a1857072
20a151274a2584175a9841e19bbd3098315c8868
df1466b4ddae4c1c9d8b385332a800eeb7f28049
58fca823e2dbb0c94774cc5f9a970c165deffeff
```